src/pybind/mgr/volumes/module.py

```diff
@@ -11,6 +11,7 @@
 import orchestrator
 
 from .fs.subvolume import SubvolumePath, SubvolumeClient
+import volume
```

Comment:
I prefer if this module was fs.volume, like fs.subvolume. We might in the future have RBD volumes module.